test/ctia/test_helpers/es.clj

```diff
@@ -88,18 +88,24 @@
   (doseq [entity (keys (all-stores))]
     (purge-index-and-template entity services)))
 
+(defn -es-port []
+  (if ((h/get-es-versions-to-test) 5) "9205" "9207"))
```

Comment:
How about a more explicit variant?

```suggestion
  (if (contains? (h/get-es-versions-to-test) 5) "9205" "9207"))
```

and does it need to be a function?

Reply:
Seemed useful to be able to set the `ctia.test.es-versions` system property at runtime, so I made it a function.

Reply:
I like the interface of a set, so I renamed then fn `set-of-es-versions-to-test`.